### PR TITLE
Update python sample for firefox

### PR DESCRIPTION
**Pull request by adithyab94**
Added python sample for setting FireFoxOptions and Setting a custom profile for FireFox

### CODE REVIEW

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md [hunk 1]
```diff
@@ -17,7 +17,10 @@ options.addPreference("network.proxy.type", 0);
 driver = new RemoteWebDriver(options);
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# We don't have a Python code sample yet -  Help us out and raise a PR
+from selenium.webdriver.firefox.options import Options
+options = Options()
+options.add_argument("--headless")
```
Comment (cgoldberg) [inline docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md]: we should show use of the `headless` property instead... like this: 
`options.headless = True`

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md [hunk 2]
```diff
@@ -43,7 +46,12 @@ options.setProfile(profile);
 driver = new RemoteWebDriver(options);
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# We don't have a Python code sample yet -  Help us out and raise a PR
+from selenium.webdriver.firefox.options import Options
+from selenium.webdriver.firefox.firefox_profile import FirefoxProfile
+options=Options()
+firefox_profile = FirefoxProfile(profile)
```
Comment (lmtierney) [inline docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md]: No need to create a `FirefoxProfile` to give to options, it does it for you if you pass a string.

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md [hunk 3]
```diff
@@ -43,7 +46,12 @@ options.setProfile(profile);
...
+options.profile(firefox_profile)
```
Comment (lmtierney) [inline docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md]: `options.profile` is a setter, not a method. This should be `options.profile = profile`

**Review by lmtierney — CHANGES_REQUESTED** [review]

**Review by diemol — CHANGES_REQUESTED** [review]
@adithyab94 could you please address the comments so we can go ahead and merge this PR? Thanks!

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md (hunk 1 above)
Comment (adithyab94) [reply docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md]: Thanks for comment. I have updated the code with above suggestion

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md (hunk 3 above)
Comment (adithyab94) [reply docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md]: Thanks for the comment. Have updated the code with above suggestions

**Review by diemol — CHANGES_REQUESTED** [review]
@adithyab94 Could you please update your fork and solve the merge conflicts please?

**Comment by adithyab94** [comment]
Updated the above changes in new pull request - #291

**Comment by adithyab94** [comment]
> Updated the above changes in new pull request - #291 

This pull request was raised from browser so it become quite messy. So created a new pull request-#291

Sorry for inconvenience

**Comment by diemol** [comment]
Replaced by #291

**Pull request closed without merging**